include/spirv-tools/libspirv.h

```diff
@@ -351,6 +351,10 @@ typedef enum {
 // Returns a string describing the given SPIR-V target environment.
 const char* spvTargetEnvDescription(spv_target_env env);
 
+// Parses s into *env and returns true if successful.  If unparsable, returns
+// false and sets *env to SPV_ENV_UNIVERSAL_1_0.
+bool spvParseTargetEnv(const char* s, spv_target_env* env);
```

Comment:
Do we really need to expose this function in the public API? Other public APIs in `libspirv.h` use `spv_target_env` directly; I don't see the points to expose a function converting strings to `spv_target_env`s here. It's mainly for our internal use.